<?xml version='1.0' encoding='UTF-8'?>
<root>
 <tabbed_widget name="Main Window" parent="main_window">
  <plotmatrix rows="2" tab_name="geometric" columns="1">
   <plot row="0" col="0">
    <range right="24.993200" top="0.057519" left="0.000000" bottom="-0.013619"/>
    <limitY/>
    <curve custom_transform="noTransform" G="7" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BI/x" B="255" R="0"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BI/y" B="17" R="255"/>
    <curve custom_transform="noTransform" G="131" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BI/z" B="17" R="0"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="0">
    <range right="24.993200" top="0.044852" left="0.000000" bottom="-0.065227"/>
    <limitY/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BP/x" B="255" R="2"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BP/y" B="0" R="255"/>
    <curve custom_transform="noTransform" G="150" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BP/z" B="2" R="0"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <plotmatrix rows="2" tab_name="inertia" columns="2">
   <plot row="0" col="0">
    <range right="24.993200" top="0.030076" left="0.000000" bottom="-0.016610"/>
    <limitY/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BM/x" B="255" R="0"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BM/y" B="3" R="255"/>
    <curve custom_transform="noTransform" G="138" name="/quadrotor/geom_inertia_estimator/param_estimates/r_BM/z" B="4" R="0"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="0">
    <range right="24.993200" top="0.001161" left="0.000000" bottom="0.000522"/>
    <limitY/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/I/x" B="255" R="0"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/I/y" B="0" R="255"/>
    <curve custom_transform="noTransform" G="90" name="/quadrotor/geom_inertia_estimator/param_estimates/I/z" B="0" R="3"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="0" col="1">
    <range right="24.993200" top="0.441239" left="0.000000" bottom="0.230776"/>
    <limitY/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/m" B="0" R="0"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="1">
    <range right="0.000000" top="1000.000000" left="0.000000" bottom="0.000000"/>
    <limitY/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <plotmatrix rows="2" tab_name="trajectory" columns="2">
   <plot row="0" col="0">
    <range right="24.993200" top="1.481425" left="0.000000" bottom="-0.434781"/>
    <limitY/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/r/x" B="255" R="0"/>
    <curve custom_transform="noTransform" G="3" name="/quadrotor/geom_inertia_estimator/param_estimates/r/y" B="0" R="255"/>
    <curve custom_transform="noTransform" G="121" name="/quadrotor/geom_inertia_estimator/param_estimates/r/z" B="7" R="0"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="0">
    <range right="24.993200" top="1.035622" left="0.000000" bottom="-0.466662"/>
    <limitY/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/q/w" B="0" R="0"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/q/x" B="255" R="5"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/q/y" B="15" R="255"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/q/z" B="0" R="0"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="0" col="1">
    <range right="24.993200" top="1.825801" left="0.000000" bottom="-1.676190"/>
    <limitY/>
    <curve custom_transform="noTransform" G="11" name="/quadrotor/geom_inertia_estimator/param_estimates/v/x" B="255" R="0"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/v/y" B="20" R="255"/>
    <curve custom_transform="noTransform" G="137" name="/quadrotor/geom_inertia_estimator/param_estimates/v/z" B="2" R="0"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="1">
    <range right="24.993200" top="7.785002" left="0.000000" bottom="-8.173232"/>
    <limitY/>
    <curve custom_transform="noTransform" G="23" name="/quadrotor/geom_inertia_estimator/param_estimates/Omega/x" B="255" R="0"/>
    <curve custom_transform="noTransform" G="3" name="/quadrotor/geom_inertia_estimator/param_estimates/Omega/y" B="0" R="255"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/Omega/z" B="3" R="0"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <plotmatrix rows="2" tab_name="biases" columns="1">
   <plot row="0" col="0">
    <range right="24.993200" top="1.529774" left="0.000000" bottom="-0.546600"/>
    <limitY/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/b_a/x" B="255" R="0"/>
    <curve custom_transform="noTransform" G="13" name="/quadrotor/geom_inertia_estimator/param_estimates/b_a/y" B="0" R="255"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/b_a/z" B="0" R="0"/>
    <transform value="noTransform"/>
   </plot>
   <plot row="1" col="0">
    <range right="24.993200" top="0.150850" left="0.000000" bottom="-0.175033"/>
    <limitY/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/b_Omega/x" B="255" R="0"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/b_Omega/y" B="7" R="255"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/b_Omega/z" B="0" R="13"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <plotmatrix rows="1" tab_name="covariances" columns="1">
   <plot row="0" col="0">
    <range right="24.993200" top="5.126984" left="0.000000" bottom="-0.125048"/>
    <limitY/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.0" B="128" R="0"/>
    <curve custom_transform="noTransform" G="160" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.1" B="164" R="160"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.10" B="0" R="128"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.11" B="0" R="128"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.12" B="255" R="0"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.13" B="0" R="0"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.14" B="0" R="255"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.15" B="255" R="255"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.16" B="128" R="0"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.17" B="128" R="0"/>
    <curve custom_transform="noTransform" G="160" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.18" B="164" R="160"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.19" B="0" R="128"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.2" B="0" R="128"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.20" B="255" R="0"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.21" B="0" R="0"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.22" B="0" R="255"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.23" B="255" R="255"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.24" B="128" R="0"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.25" B="128" R="0"/>
    <curve custom_transform="noTransform" G="160" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.26" B="164" R="160"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.27" B="0" R="128"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.28" B="0" R="128"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.29" B="255" R="0"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.3" B="0" R="0"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.4" B="0" R="255"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.5" B="255" R="255"/>
    <curve custom_transform="noTransform" G="0" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.6" B="128" R="0"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.7" B="128" R="0"/>
    <curve custom_transform="noTransform" G="160" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.8" B="164" R="160"/>
    <curve custom_transform="noTransform" G="128" name="/quadrotor/geom_inertia_estimator/param_estimates/covariance.9" B="0" R="128"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <currentPlotMatrix index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <Plugins>
  <DataLoad_CSV>
   <default time_axis=""/>
  </DataLoad_CSV>
  <DataLoad_ROS_bags>
   <selected_topics list=""/>
  </DataLoad_ROS_bags>
  <DataLoad_ULog>
   <no_params/>
  </DataLoad_ULog>
  <ROS_Topic_Streamer>
   <selected_topics list="/quadrotor/geom_inertia_estimator/param_estimates"/>
  </ROS_Topic_Streamer>
  <RosoutPublisherROS/>
  <TopicPublisherROS/>
 </Plugins>
 <previouslyLoadedStreamer name="ROS_Topic_Streamer"/>
</root>

